### ISSUE TITLE
Extreme slowness linting a function with documentation

### ISSUE THREAD

**Issue by alcarvalho** [realm/SwiftLint#755]
I'm experiencing an extreme slowness linting the following file, on version `0.11.1`

``` swift
extension SequenceType {

    /// Returns the result of repeatedly calling `combine` with an
    /// accumulated value initialized with the first and each element of
    /// `self`, in turn
    @warn_unused_result
    public func reduce(@noescape combine: (Self.Generator.Element, Self.Generator.Element)
        throws -> Self.Generator.Element) rethrows -> Self.Generator.Element? {
        var generator = generate()
        var last = generator.next()
        var next: Self.Generator.Element?
        repeat {
            next = generator.next()
            if let _last = last,
                _next = next {
                last = try combine(_last, _next)
            }
        } while next != nil

        return last
    }
}
```

On my MacBook Pro with 2.5 GHz Intel Core i7 this is the time it takes:

```
Linting Swift files in current working directory
Linting 'SequenceType.swift' (1/1)
Done linting! Found 0 violations, 0 serious in 1 file.
      107.86 real       107.39 user         0.35 sys
```

If I remove the documentation:

```
Linting Swift files in current working directory
Linting 'SequenceType.swift' (1/1)
Done linting! Found 0 violations, 0 serious in 1 file.
        0.03 real         0.01 user         0.00 sys
```

**Comment by jpsim** [comment]
Wow, two minutes to lint! This is crazy. Will label this as a bug.

**Comment by marcelofabri** [comment]
Is this still an issue, @alcarvalho?

**Comment by alcarvalho** [comment]
@marcelofabri no. I just tested this sample on 0.17.0 and it's lightning fast, now.

**Comment by marcelofabri** [comment]
Thanks for the feedback!
